pkg/controller/nodeipam/ipam/range_allocator.go

```diff
@@ -62,12 +62,15 @@ type rangeAllocator struct {
 	nodesInProcessing sets.String
 }
 
+var _ CIDRAllocator = (*rangeAllocator)(nil)
```

Comment:
The common way to write this is
```suggestion
var _ CIDRAllocator = &rangeAllocator{}
```